### PR TITLE
Add more JVMs to CI

### PR DESCRIPTION
**Pull request by durban**
This is just testing, because something doesn't seem to work with certain JVM versions.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,8 +14,12 @@ ThisBuild / startYear := Some(2022)
 ThisBuild / githubWorkflowJavaVersions ++= Seq(
   JavaSpec.temurin("11"),
   JavaSpec.temurin("17"),
+  JavaSpec.graalvm("11"),
```
Comment (armanbilge) [inline build.sbt:17]: I'm not sure if GraalVM 11 is available like this. I think you need to specify it the "old" way.

https://github.com/typelevel/sbt-typelevel/blob/7a9074633fbb4d3fcf804f246c9f269885a5b35b/github-actions/src/main/scala/org/typelevel/sbt/gha/JavaSpec.scala#L43

Comment (armanbilge) [reply build.sbt:17]: Also is GraalVM for JDK 11 still relevant?

Comment (durban) [reply build.sbt:17]: `JavaSpec.graalvm("11")` seems to be the same as `JavaSpec(Distribution.GraalVM, "11")`.

Comment (armanbilge) [reply build.sbt:17]: Right, I think you need to do something like `JavaSpec(Distribution.GraalVM("21.3.0"), "11")`. This is the old versioning scheme, they changed it for the latest releases.

Comment (armanbilge) [reply build.sbt]: Ah, I see this is still broken 😅

Comment (durban) [reply build.sbt]: Yeah, apparently the generated yaml is invalid...

**Pull request closed without merging**